hack/lib/util/environment.sh

```diff
@@ -148,7 +149,17 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    mkdir -p  "${BASETMPDIR}" "${LOG_DIR}" "${VOLUME_DIR}" "${ARTIFACT_DIR}" "${HOME}"
+    # ensure that the directories are clean
+    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
```

Comment:
We were doing the before, as well if we had issues removing the directory, so if you never had `findmnt` it should not change behavior for you. Is `findmnt` not available on OSX?